docs/_posts/2022-06-03-angular-change-detection.md

```diff
@@ -1,39 +1,38 @@
 ---
 layout: post
-title: "Angular Changedetection Playground"
+title: "Angular change detection Playground"
```

Comment:
I have to disagree. Have a look here: https://blog.angular-university.io/how-does-angular-2-change-detection-really-work/#:~:text=Angular%20change%20detection%20is%20a,and%20its%20HTML%20template%20view. If you are talking about the `ChangeDetectorRef` the capitalisation would apply. Since you are—presumably—talking about the general concept, rather than an Angular component, it should be "change detection" imo.